[W09-2] Health Hub

Lookuz opened the pull request.

marlenekoh reviewed (commented):
@Lookuz @huichun66 @brohandev @daviddl9  @jing1324 
### README
✅ Updated UI.png
✅ Updated description
✅ Updated travis, coveralls, appveyor badges

### AboutUs
✅ Photos uploaded
✅ Roles and components updated

### User Guide
✅ Intro section updated
✅ Features updated
(More comments given separately in PDF)

### Developer Guide
✅ Product scope updated
❌ Please update the Implementation section with your commands
Unnecessary sections such as Appendix A and G can be removed (:
(More comments given separately in PDF)

Remember to update the following with your team name (:
> By: Team SE-EDU Since: Jun 2016 Licence: MIT

Lookuz committed:
Modified User Guide for additional clarity. Added description to edit command in Developer Guide.

Lookuz committed:
Merge branch 'master' into Lukaz-v1.3

Lookuz committed:
Merge pull request #161 from Lookuz/Lukaz-v1.3

Addition to Developer Guide

Lookuz committed:
Fixed filter healthworker bug. Added use of substring to filtering by skills for healthworker.

Lookuz committed:
Fixed bugs in naming convention.

Lookuz committed:
Merge remote-tracking branch 'origin/Lukaz-v1.3' into Lukaz-v1.3

# Conflicts:
#	docs/UserGuide.adoc

jing1324 committed:
Merge pull request #201 from Lookuz/Lukaz-v1.3

Fixed bug in FilterHealthWorkerCommand

merody66 committed:
Merge pull request #36 from CS2103-AY1819S2-W09-2/master

Merge latest changes into local fork (070419)

daviddl9 committed:
Added tags for RepoSense compatibility

daviddl9 committed:
Merge pull request #202 from daviddl9/add_find_request_command

Added author tags for RepoSense compatibility

daviddl9 committed:
Updated config file to add author name

daviddl9 committed:
Merge pull request #203 from daviddl9/add_find_request_command

Updated reposense config file to add author name

daviddl9 committed:
Fixed EditRequestCommand bug and added documentation

daviddl9 committed:
Updated user guide

daviddl9 committed:
Fixed documentation issue with assign command

jing1324 committed:
BugFix: Unable to undo a complete command

daviddl9 committed:
Updated clear and edit request command tests

jing1324 committed:
Fix bug in undo unable to undo completeRequest

jing1324 committed:
Merge pull request #204 from daviddl9/add_find_request_command

Fixed edit request command bug

jing1324 committed:
Fix bug with CompleteRequestCommand test

jing1324 committed:
Fix new line at EOF

jing1324 committed:
Fix new line at EOF

daviddl9 committed:
Updated dev guide diagrams

daviddl9 committed:
Merge pull request #206 from daviddl9/add_find_request_command

Updated dev guide diagrams

merody66 committed:
Merge pull request #37 from CS2103-AY1819S2-W09-2/master

Merge latest changes into local fork (090419)

daviddl9 committed:
Updated user guide

merody66 committed:
Merge pull request #205 from jing1324/bugfix

Fixed Bug with Undo Command unable to undo a Complete Command

merody66 committed:
Merge pull request #38 from CS2103-AY1819S2-W09-2/master

Merge latest changes.

brohandev committed:
Merge pull request #17 from CS2103-AY1819S2-W09-2/master

090419

daviddl9 committed:
Updated documentation

daviddl9 committed:
updated healthworker book

daviddl9 committed:
Merge pr from org

brohandev committed:
fixed statistics class
1) initializes with the correct conditions values at startup time
2) inegrated undo redo features
3) removes condition from statistics list completely upon reaching 0 occurences

merody66 committed:
Modified the command usage message for clarity.

merody66 committed:
Renamed AddressBookSystemTest to HealthHubSystemTest, and implemented SampleDataTest.

brohandev committed:
checkstyle

merody66 committed:
Fix checkstyle.

brohandev committed:
checkstyle

brohandev committed:
checkstyle

brohandev committed:
Merge pull request #209 from huichun66/master

Refractored AddressBookSystemTest and implemented SampleDataTest.

merody66 committed:
Merge pull request #208 from brohandev/master

Revamped Statistics Class

daviddl9 committed:
Changed file names

daviddl9 committed:
resolved conflict

daviddl9 committed:
resolved conflict

daviddl9 committed:
Merge pull request #207 from daviddl9/add_find_request_command

Added user guide documentation

daviddl9 committed:
Updated images and resized images

daviddl9 committed:
Merge pull request #210 from daviddl9/add_find_request_command

Updated UserGuide and DeveloperGuide

daviddl9 committed:
Updated UG

daviddl9 committed:
Merge pull request #211 from daviddl9/add_find_request_command

Updated UG

daviddl9 committed:
Added logger to assignrequestcommand

Lookuz committed:
Updated UG filter healthworker command with image example.

Lookuz committed:
Fixed issue with image not rendering.

Lookuz committed:
Added developer guide contributions to PPP.

merody66 committed:
Renamed PPP file name.

merody66 committed:
Merge pull request #213 from huichun66/master

Renamed PPP file name.

Lookuz committed:
Added sequence diagrams for filter in Developer Guide. Restructured some components in the Developer Guide.

Lookuz committed:
Merge branch 'master' into Lukaz-v1.3

jing1324 committed:
Modified User and Developer guides

merody66 committed:
Merge pull request #39 from CS2103-AY1819S2-W09-2/master

Merge latest changes into local fork (120419)

Lookuz committed:
Implemented isAssigned methods in ModelManager, RequestBook and UniqueRequestList.

Lookuz committed:
Fixed EOF in config.json and healthworkerbook.json

Lookuz committed:
Merge remote-tracking branch 'origin/Lukaz-v1.3' into Lukaz-v1.3

# Conflicts:
#	data/healthworkerbook.json

Lookuz committed:
Fixed minor bugs in isAssigned tests.

Lookuz committed:
Added bug fix to prevent deletion of HealthWorker that are assigned to pending requests.

daviddl9 committed:
Merge pull request #214 from Lookuz/Lukaz-v1.3

Lucas: Changes to Developer Guide & Updated DeleteHealthWorkerCommand

daviddl9 committed:
merge pr from org

daviddl9 committed:
Merged updates from org

Lookuz committed:
Added functionality to filter healthworker by nric field.

Lookuz committed:
Added methods for updating request upon updating health worker.

Lookuz committed:
Amending single-space line break in lookuz.adoc

daviddl9 committed:
Merged updates from org

daviddl9 committed:
Updated assign request to check for conflicting timings of healthworkers

daviddl9 committed:
Fixed edit request command bug

daviddl9 committed:
Updated user guide

daviddl9 committed:
Added minor documentation

merody66 committed:
Changed huichun.adoc to huichun66.adoc for reference.

merody66 committed:
Uncommented ModelHelper.java

merody66 committed:
Uncommented some lines in TestApp.java

merody66 committed:
Uncommented some lines in TestUtil.java and implemented some methods in InfoPanelHandle.java

merody66 committed:
Edited TypicalRequests and TypicalHealthWorkers sample data to be a bit realistic.

merody66 committed:
Implemented HelpCommandSystemTest.java

merody66 committed:
Implemented SelectCommandSystemTest.java

merody66 committed:
Fixed checkstyle errors.

merody66 committed:
Modified HealthHubSystemTest.java and implemented DeleteRequestCommandSystemTest.java

merody66 committed:
Merge branch 'master' into master

merody66 committed:
Merge pull request #40 from CS2103-AY1819S2-W09-2/master

Merge latest changes into local fork (130419)

merody66 committed:
Modified typicalrequests and typicalhealthworkers json files for consistency.

merody66 committed:
Fixed some of the failed tests.

merody66 committed:
Modified typicalrequests.java

merody66 committed:
Merge pull request #217 from huichun66/master

PR to fix my previous commit.

daviddl9 committed:
Rectified merge conflict

daviddl9 committed:
added eof to config

merody66 committed:
Merge pull request #216 from daviddl9/add_find_request_command

Fix assign request command bug

Lookuz committed:
Updated Diagrams for DeveloperGuide.

Lookuz committed:
Merge branch 'master' into Lukaz-v1.3

# Conflicts:
#	data/healthworkerbook.json
#	src/main/java/seedu/address/logic/commands/DeleteHealthWorkerCommand.java
#	src/main/java/seedu/address/model/ModelManager.java
#	src/test/java/seedu/address/TestApp.java

Lookuz committed:
Added automatic update of assigned healthworker NRIC when NRIC field in HealthWorker class is edited.

Lookuz committed:
Fixed EOF in json files

Lookuz committed:
Fixed bug in AddRequestCommandTest

Lookuz committed:
Updated TestApp to include typical requests and healthworkers

Lookuz committed:
Updated System Tests.

Lookuz committed:
Fixed indentation errors.

Lookuz committed:
Stupid asciidoctor bugs.

Lookuz committed:
FUCK LA LOL

Lookuz committed:
Deprecated tests that are not working.

Lookuz committed:
Javadocs

merody66 committed:
Merge pull request #41 from CS2103-AY1819S2-W09-2/master

Merge latest changes into local fork (130419)

Lookuz committed:
config.json

Lookuz committed:
I HEAR THE DRUMS ECHOING TONIGHT

jing1324 committed:
Resolve merge conflict

merody66 committed:
Merge pull request #215 from Lookuz/Lukaz-v1.3

Lucas: Improvements to FilterCommandParser and EditHealthWorkerCommand

merody66 committed:
Merge pull request #42 from CS2103-AY1819S2-W09-2/master

Merge latest changes into local fork (140419)

jing1324 committed:
Modify some diagrams in DG

merody66 committed:
Renamed FindCommandSystemTest.java to FilterCommandSystemTest.java

merody66 committed:
Fixed SampleDataTest problems.

merody66 committed:
Commented out JsonSerializableRequestBookTest.java, behaves very irratic.

merody66 committed:
Uncommented SampleDataTest.java and implemented FilterCommandSystemTest.java

merody66 committed:
Added nric field to show in the GUI for health worker.

merody66 committed:
Fix checkstyle errors.

merody66 committed:
Renamed some methods.

merody66 committed:
Implemented FilterCommandSystemTest.java

merody66 committed:
Renamed FilterCommandSystemTest.java to FilterRequestCommandSystemTest.java

merody66 committed:
Updated some minor css.

merody66 committed:
Changed UI image to an updated one.

merody66 committed:
Updated Ui.png to show more details.

merody66 committed:
Added a line of code to fix a bug.

daviddl9 committed:
Merge pull request #219 from huichun66/master

Implemented FilterRequestCommandSystemTest + add NRIC field on UI.

brohandev committed:
Merge pull request #18 from CS2103-AY1819S2-W09-2/master

140419

jing1324 committed:
Modify Storage diagram

jing1324 committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S2-W09-2/main into documentation

brohandev committed:
editRequest_statistics bug fixed

jing1324 committed:
Touch up on DG

jing1324 committed:
Add version 1.4 release notes to UG

brohandev committed:
implemented ordering by value of treemap into a linkedhashmap

brohandev committed:
javadocs

jing1324 committed:
Modify PPP

jing1324 committed:
Update config.json

jing1324 committed:
Hopefully this solved the EOF problem

jing1324 committed:
Merge pull request #220 from brohandev/master

Statistics Improvements

jing1324 committed:
Merge pull request #218 from jing1324/documentation

Ready for review- Update of docs

brohandev committed:
user guide

jing1324 committed:
Commit to allow checkout

brohandev committed:
Merge pull request #221 from brohandev/master

user guide (statistics)

jing1324 committed:
Added new CommandType to facilitate undo/redo of both books at once

jing1324 committed:
Implement undo/redo for new EditHealthWorkerCommand

jing1324 committed:
Fix some bugs

Lookuz committed:
Shortened PPP by cutting out redundant information

Lookuz committed:
Improved elaboration in certain portions of the Developer Guide.

Lookuz committed:
Fixed various portions of the Developer Guide

Lookuz committed:
Merge pull request #222 from Lookuz/Lukaz-v1.3

Lucas: Updating PPP and Developer Guide

Lookuz committed:
Modified PPP slightly.

Lookuz committed:
Merge pull request #223 from Lookuz/Lukaz-v1.3

Modified PPP slightly.

jing1324 committed:
Reimplement Tests involving undo redo

jing1324 committed:
Fix checkstyle errors

jing1324 committed:
Fix EOF

daviddl9 committed:
Updated PPP

jing1324 committed:
Update user guides and small bug fixes

jing1324 committed:
Fix merge conflicts

jing1324 committed:
Fix EOF

daviddl9 committed:
merge pr from org

daviddl9 committed:
Updated PPP

jing1324 committed:
Commented out unstable test

daviddl9 committed:
Merge pull request #224 from jing1324/undoRedo

Reimplementation of Undo redo

daviddl9 committed:
fixed checkstyle

jing1324 committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S2-W09-2/main into documentation

daviddl9 committed:
Merge pull request #225 from jing1324/documentation

Update user guides and small bug fixes

jing1324 committed:
Fix bug with test

daviddl9 committed:
merge pr from org

daviddl9 committed:
Updated images and documentation

daviddl9 committed:
Merge pull request #227 from jing1324/documentation

Bug fix on tests

daviddl9 committed:
merge pr from org

daviddl9 committed:
Updated dev documentation images

brohandev committed:
streamlined more statistics commands

brohandev committed:
checkstyle unused imports

brohandev committed:
developer guide gains

merody66 committed:
Merge pull request #228 from brohandev/master

streamlined more statistics commands

merody66 committed:
Merge pull request #43 from CS2103-AY1819S2-W09-2/master

Merge latest changes into local fork (150419)

merody66 committed:
Updated UiClassDiagram.png

merody66 committed:
Updated UserGuide.adoc

merody66 committed:
Merge pull request #229 from huichun66/master

Updated UiClassDiagram and UserGuide

merody66 committed:
Updated some lines in UserGuide.adoc

merody66 committed:
Updated my PPP.

merody66 committed:
Merge pull request #230 from huichun66/master

Updated my PPP.

jing1324 committed:
Small changes to Developer guide

jing1324 committed:
Add reposense to PPP

brohandev committed:
Merge pull request #231 from brohandev/master

DG statistics gains

jing1324 committed:
Modified sequence diagram for dg

daviddl9 committed:
Updated dev guide implementation for assignrequest

brohandev committed:
UG gainzzzzzz

daviddl9 committed:
Added tests for JsonAdaptedRequestTest

daviddl9 committed:
Merge pull request #226 from daviddl9/add_find_request_command

Updated PPP; Added JsonAdaptedRequestTests

jing1324 committed:
Modify parts of dev

brohandev committed:
brohandev ppp

brohandev committed:
Merge pull request #232 from brohandev/master

UG gainzzzzzz

jing1324 committed:
Final changes to ug dg and ppp

brohandev committed:
brohandev GAINZZZZ

jing1324 committed:
Merge pull request #233 from jing1324/documentation

Final changes to documentation

brohandev committed:
Merge pull request #234 from brohandev/master

brohandev ppp GAINZZZ

brohandev committed:
cosmetic changes

brohandev committed:
further changes

brohandev committed:
more

merody66 committed:
Implemented ClearCommandSystemTest.java

merody66 committed:
Added my author tag hahaha

brohandev committed:
Merge pull request #235 from brohandev/master

but wait there's more

daviddl9 committed:
Updated UG documentation

daviddl9 committed:
Merge pull request #236 from daviddl9/add_find_request_command

Updated UG documentation

merody66 committed:
Added SetReminderCommandSequenceDiagrams.

merody66 committed:
Added helper images.

merody66 committed:
Updated docs.

merody66 committed:
Fix checkstyle errors.

merody66 committed:
Merge pull request #237 from huichun66/master

Implemented ClearCommandSystemTest and updated docs.

merody66 committed:
Renamed file extensions.

merody66 committed:
Merge pull request #238 from huichun66/master

Renamed file extensions.

daviddl9 committed:
Edited PPP

jing1324 committed:
Add more sampleutildata for testing

jing1324 committed:
Fix checkstyle errors

jing1324 committed:
Fix eof

jing1324 committed:
Fix image unable to display in DG

daviddl9 committed:
Merge pull request #239 from jing1324/master

Add more sampleutildata for testing

daviddl9 committed:
merge pr from org

merody66 committed:
Updated docs and added AutoCompleteTextFieldSMD.png

merody66 committed:
Merge pull request #240 from huichun66/master

Updated docs and added AutoCompleteTextFieldSMD.png

merody66 committed:
Merge pull request #44 from CS2103-AY1819S2-W09-2/master

Merge latest changes to local fork (150419)

merody66 committed:
Updated Ui_home.png and uploaded a new Ui_home image to use for my PPP.

merody66 committed:
Updated PPP and developer guide.

merody66 committed:
Fixed .adoc errors

merody66 committed:
Merge pull request #241 from huichun66/master

Updated PPP.

merody66 committed:
Renamed file extension of image.

merody66 committed:
Merge pull request #242 from huichun66/master

Renamed file extension of image.

merody66 committed:
Updated PPP to reduce pages.

merody66 committed:
Updated PPP.

merody66 committed:
Merge pull request #243 from huichun66/master

Updated my PPP.

merody66 committed:
Updated developer guide.

merody66 committed:
Merge pull request #244 from huichun66/master

Updated developer guide.

daviddl9 committed:
Fixed tests and updated User Guide

daviddl9 committed:
merge pr from org

Lookuz committed:
Slight changes to the Developer Guide.

Lookuz committed:
Merge pull request #245 from Lookuz/Lukaz-v1.3

Slight changes to the Developer Guide.

daviddl9 committed:
Merge pull request #246 from daviddl9/add_find_request_command

Updated UG and fixed minor bug

brohandev committed:
reposense

brohandev committed:
Merge pull request #19 from CS2103-AY1819S2-W09-2/master

150419

brohandev committed:
Merge pull request #247 from brohandev/master

reposense

daviddl9 committed:
Update daviddl9.adoc

daviddl9 committed:
Revert "reposense"

daviddl9 committed:
Merge pull request #248 from CS2103-AY1819S2-W09-2/revert-247-master

Revert "reposense"

brohandev committed:
Merge pull request #20 from CS2103-AY1819S2-W09-2/master

220419

brohandev committed:
reposense

brohandev committed:
reposense

daviddl9 committed:
Merge pull request #249 from brohandev/master

reposense to detect author contributions

brohandev committed:
reposense

brohandev committed:
Merge pull request #250 from brohandev/master

reposense
